pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/column/DefaultNullValueVirtualColumnProvider.java

```diff
@@ -88,6 +88,11 @@ public ColumnMetadataImpl buildMetadata(VirtualColumnContext context) {
     ColumnMetadataImpl.Builder builder = getColumnMetadataBuilder(context).setCardinality(1).setHasDictionary(true);
     if (context.getFieldSpec().isSingleValueField()) {
       builder.setSorted(true);
+
+      if (context.getFieldSpec().getDefaultNullValue() instanceof Comparable) {
+        builder.setMinValue((Comparable) context.getFieldSpec().getDefaultNullValue());
+        builder.setMaxValue((Comparable) context.getFieldSpec().getDefaultNullValue());
+      }
     } else {
       // When there is no value for a multi-value column, the maxNumberOfMultiValues and cardinality should be
```

Comment:
We want to set the same min & max value for the MV column

Reply:
Ack. Added for all column types